x/wasm/types/authz.pb.go

```diff
@@ -5,25 +5,22 @@ package types
 
 import (
 	fmt "fmt"
-	io "io"
```

Comment:
Can you run `make format` to have the imports formatted well?
fyi: `make proto-all` should be the preferred target for generation. It includes linter and formatting

Reply:
:) sounds great